nengo_loihi/loihi_api.py

```diff
@@ -99,6 +99,46 @@ def tracing_mag_int_frac(synapses):
     return mag_int, mag_frac
 
 
+def decay_int(x, decay, bits=12, offset=0, out=None):
+    """Decay integer values using a decay constant.
+
+    The decayed value is given by
+
+        sign(x) * floor(abs(x) * (2**bits - offset - decay) / 2**bits)
```

Comment:
This will render as a blockquote, which is probably not what you want. If you want it to show up like math, use the `.. math::` directive; if you want it to look like code you can either end the previous line with `::` or use the `.. code-block::` directive; see https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

Reply:
Can we talk about standards for how we do equations? I find LaTeX great for stuff that will definitely be rendered, but these docstrings may either be rendered (for online) or just as strings (when reading the code, or calling `help` on a function).

For that reason, I think that we should stay away from the `math` directive, unless we have a really good reason for it in a particular situation. I'd opt for using `::` and displaying stuff as code wherever possible, since this is clear in either raw or rendered format.

Reply:
Having said that, I realize that some things are hard to do in code (or at least need a standard way of writing them), for example a subscript like `x_{i-1}`. Sums are another big one that I tend to write in LaTeX notation. So I'm not sure which way to go on this.